src/pages/listing/Listing.tsx

```diff
@@ -20,7 +18,7 @@ const Listing = ({ match }: RouterProps) => (
         <ListingPage {...data.listing} />
       }
     </Query>
-    <Footer />
+    <DetailedFooter className="d-none d-lg-block" />
```

Comment:
actually now that `<DetailedFooter>` takes a classname, i think you can just export the default `<Footer>` and use react router to handle the all the footers again

Reply:
🙏